logs.rst

```diff
@@ -69,3 +69,15 @@ Results for Pruning Algorithms -
 |            |         +---------------+----------------+-----------------+----------+
 |            |         |      3/3      |      18.99     |        5        |  98.90 % |
 +------------+---------+---------------+----------------+-----------------+----------+
+
+Results for  Dynamic Quantization - 
+===================================
+
++------------+---------+---------------+----------------+-----------------+----------+--------------+
+| Model Used | Dataset |    Model      |    Size (MB)   | Training epochs | Accuracy | Time elapsed |
++============+=========+===============+================+=================+==========+==============+
+|            |         |     Base      |       4.15     |                 |  0.9582  |  28.84 s     |
+|            |         +---------------+----------------+                 +----------+--------------+
+|  ResNet152 |  MNIST  |   Quantized   |       0.56     |        1        |  0.9583  |  28.71 s     |
++------------+---------+---------------+----------------+-----------------+----------+--------------+
```

Comment:
Can you explain the columns and the whole process that you followed once?

Reply:
I don't understand the time elapsed column for example

Reply:
Oh sure.
Process: I trained a ResNet152 for one epoch on MNIST. After training it was quantized. The accuracy obtained by the non-quantized model was 0.9582 and it took 28.84 secs for it to run on the test set on CPU. The quantized model achieved an accuracy of 0.9583 and it took 28.71 secs for it to run on the test set on CPU.
Time elapsed is the time the model took to run on the test set.

Here is a link to a similar analysis by PyTorch:
https://pytorch.org/tutorials/advanced/dynamic_quantization_tutorial.html

Reply:
How many epochs did you train ResNet for?

Reply:
One epoch

Reply:
Can you perform a similar analysis for CIFAR-10 with more training epochs maybe?

Reply:
I wouldn't mind but why is that needed?